modules/developers-guide/pages/tutorials/define-an-actor.adoc

```diff
@@ -92,7 +92,7 @@ Let's take a closer look at this simple program:
 +
 --
 * The program imports a `+Debug+` module to provide the `+print+` functionality.
-* The program uses the `+public query func+` to define a query method because, in this case, the `+actor_hello+` program doesn't make any changes to the state of the canister or perform any operations that would update the data you are accessing.
+* The program uses the `+public query func+` to define a query method because, in this case, the `+actor_hello+` program doesn't make any changes to the state of the canister smart contract or perform any operations that would update the data you are accessing.
```

Comment:
```suggestion
* The program uses the `+public query func+` to define a query method because, in this case, the `+actor_hello+` program doesn't make any persistent changes to the state of the canister smart contract - any changes are transient and discarded after the query completes.
```